syntax = "proto2";
package TINYKV;
option cc_generic_services = true;
service KVService {
  rpc Get(ReqKey) returns (RespValue);
  rpc Insert(ReqKeyValue) returns (RespValue);
};
message ReqKey {
    required int64 qid = 1;
    repeated string key = 2;
};
message ReqKeyValue {
    required int64 qid = 1;
    repeated string key = 2;
    repeated string value = 3;
};

message RespValue {
    required int32 status = 1;
    repeated string value = 2;
};
